packages/block-editor/src/components/block-card/index.js

```diff
@@ -1,21 +1,35 @@
+/**
+ * WordPress dependencies
+ */
+import { Tip } from '@wordpress/components';
+
 /**
  * Internal dependencies
  */
 import BlockIcon from '../block-icon';
 
-function BlockCard( { blockType } ) {
+function BlockCard( { blockType, tip } ) {
 	return (
-		<div className="block-editor-block-card">
-			<BlockIcon icon={ blockType.icon } showColors />
-			<div className="block-editor-block-card__content">
-				<h2 className="block-editor-block-card__title">
-					{ blockType.title }
-				</h2>
-				<span className="block-editor-block-card__description">
-					{ blockType.description }
-				</span>
+		<>
+			<div className="block-editor-block-card">
+				<BlockIcon icon={ blockType.icon } showColors />
+				<div className="block-editor-block-card__content">
+					<h2 className="block-editor-block-card__title">
+						{ blockType.title }
+					</h2>
+					<span className="block-editor-block-card__description">
+						{ blockType.description }
+					</span>
+				</div>
 			</div>
-		</div>
+			{ tip && (
```

Comment:
Adding this here means potentially, the tip could show up in the inspector too. Did you really mean to add it to the BlockCard or more to the InserterPreviewPanel?

Reply:
I added it as an example of showing the scope of this approach, where it isn't limited only to the block inserter menu. There a handful of issues about Tips. [Here](https://github.com/WordPress/gutenberg/issues/17091), we'd like to show a block tip if its defined. This PR tackles it as well, allowing us registering tips by block type:

```es6
import { dispatch } from '@wordpress/data';

// Register Tips.
const { __experimentalRegisterBlockTip } = dispatch( 'core/block-editor' );
const tips = [
	__( 'Add alternative text to your images to make them more accessible.' ),
	__( 'Use a Cover block to add text on top of your image.' ),
	__( 'To place two images side by side, convert them to a Gallery.' ),
];

tips.forEach( ( desc ) => __experimentalRegisterBlockTip('core/image', desc ) );
```